nextcord/guild.py

```diff
@@ -2874,7 +2875,8 @@ async def ban(
         user: Snowflake,
         *,
         reason: Optional[str] = None,
-        delete_message_days: Literal[0, 1, 2, 3, 4, 5, 6, 7] = 1,
+        delete_message_seconds: Optional[int] = None,
```

Comment:
This type hint should be set to `int` and its default should be set to `0` to match the HTTP route implementation

Reply:
https://github.com/nextcord/nextcord/pull/813#discussion_r967581183